Doctrine/Orm/DateFilter.php

```diff
@@ -35,7 +38,7 @@ class DateFilter extends AbstractFilter
     public function __construct(ManagerRegistry $managerRegistry, array $properties = null)
     {
         $this->managerRegistry = $managerRegistry;
-        $this->properties = $properties;
+        $this->properties      = $properties;
```

Comment:
Ok, it's not significant but consistency, even in coding style is always a must :D